prime-router/src/main/kotlin/azure/HistoryFunctions.kt

```diff
@@ -258,6 +261,8 @@ open class BaseHistoryFunction {
                     .expires( it.createdAt.plusDays( DAYS_TO_SHOW ).toEpochSecond() * 1000 )
                     .facilities(facilities)
                     .actions(actions)
+                    .receivingOrg(it.receivingOrg)
+                    .receivingOrgSvc(it.receivingOrgSvc)
                 .build()        
```

Comment:
Don't we want more description than just the receiver's internal name? The receiver name is meant as an internal name.

Reply:
Since it is just an internally used name - I'm proposing to repurpose it to be the displayable name - we could always add a "displayName" to the receiver - but that would be redundant with the internal name

Reply:
There's a task on my plate to add a displayable receiver name; but I haven't gotten to it yet.

Reply:
I think the displayable name should be allowed to have spaces and other punctuation.

Reply:
done - using externalName